### PR TITLE
Notify matrix room when a PR is opened.

### PR DESCRIPTION
**Pull request by mariha**

### CODE REVIEW

File: .github/workflows/notify-on-pull-requests.yml [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
 
 on:
   pull_request:
-    types:
-      - closed
+    types: [opened, reopened, closed]
 
 jobs:
+  ping_matrix_when_opened:
+    if: github.event.pull_request.merged == false
```
Comment (mrkvon) [inline .github/workflows/notify-on-pull-requests.yml:9]: There seems to be a [bunch of events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request), and i suspect that this one will be triggered when PR is closed without merging, too. I'm not sure until we run it. Can be fixed later, i suppose

**Review by mrkvon — APPROVED** [review]
left a comment for a follow-up, otherwise LGTM. 🎉 

not sure how to test this; we need to run it and see 🙂

**Comment by mrkvon** [comment]
Oh, it actually [worked even before it was merged](https://matrix.to/#/!zgIOsIuqkYCMNrTSxP:matrix.org/$fOL44XlU-VJHxfkCYEpC28FcWKG6EU2O0eT-v8uSfGM?via=matrix.org&via=chagai.website). 😃

File: .github/workflows/notify-on-pull-requests.yml (hunk 1 above)
Comment (mariha) [reply .github/workflows/notify-on-pull-requests.yml:9]: Good point!

**Pull request merged**